frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -318,12 +311,11 @@ export const fetchCardData = createThunkAction(FETCH_CARD_DATA, function(card, d
 
 export const fetchCardDuration = createThunkAction(FETCH_CARD_DURATION, function(card, datasetQuery) {
     return async function(dispatch, getState) {
-        let result = await MetabaseApi.dataset_duration(datasetQuery);
```

Comment:
what is the response that comes back from dataset_duration look like? Are there any keys other than "average"?

Also, @tlrobinson where in the datastore does the `FETCH_CARD_DURATION` action place it's new result?